global_planner/src/astar.cpp

```diff
@@ -89,7 +89,9 @@ void AStarExpansion::add(unsigned char* costs, float* potential, float prev_pote
 
     potential[next_i] = p_calc_->calculatePotential(potential, costs[next_i] + neutral_cost_, next_i, prev_potential);
     int x = next_i % nx_, y = next_i / nx_;
-    float distance = abs(end_x - x) + abs(end_y - y);
+
+    //float distance = abs(end_x - x) + abs(end_y - y);
```

Comment:
```suggestion
```
Please just remove code instead of commenting it out.

Reply:
Sorry I'm a little new to Git and things. I would add a parameter to choose which metric to use.